samples/nrf9160/gps/src/main.c

```diff
@@ -40,6 +40,10 @@
 #endif
 #endif /* CONFIG_BOARD_THINGY91_NRF9160NS */
 
+#ifdef CONFIG_BOARD_CIRCUITDOJO_FEATHER_NRF9160NS
+#define AT_COEX0	"AT%XCOEX0=1,1,1565,1586"
```

Comment:
The percent sign could be escaped to be consistent with the other AT command string defines.
```suggestion
#define AT_COEX0	"AT\%XCOEX0=1,1,1565,1586"
```

Reply:
Done